test/assertions.test.ts

```diff
@@ -889,6 +926,181 @@ describe('runAssertion', () => {
     );
   });
 
+  // is-sql test cases:
+  it('should pass when the is-sql assertion passes', async () => {
+    const output = 'SELECT id, name FROM users';
+
+    const result: GradingResult = await runAssertion({
+      prompt: 'Some prompt',
+      provider: new OpenAiChatCompletionProvider('gpt-4'),
+      assertion: isSQLAssertion,
+      test: {} as AtomicTestCase,
+      output,
+    });
+    expect(result.pass).toBeTruthy();
+    expect(result.reason).toBe('Assertion passed');
```

Comment:
nit, there are 4 values returned on this object, write one expectation that checks all 4: 

```ts
expect(result).toEqual({ .... });

Reply:
Thank you for your suggestion. I can definitely update the expectation to check all four values in a single statement. I followed the current format because all the other test cases' expectations in this file are written this way. However, I am happy to change mine if you prefer this new format.

Reply:
Thank you for following the established conventions. Whatever you think is best. I am learning the codebase too.